modules/tensor_mechanics/test/tests/ad_simple_linear/tests

```diff
@@ -30,7 +39,17 @@
     type = 'PetscJacobianTester'
     input = 'linear-ad.i'
     run_sim = 'True'
-    prereq = 'linear-ad'
+    prereq = 'linear-ad-production'
```

Comment:
I'm wondering whether the current `linear-ad.i` and all its associated files should just be ripped out now that we have a production version. Do you see any value in keeping them around?

Reply:
hm, no, I guess we can remove it